### DISCUSSION TITLE
Emgu 4.0.30319 IOS cvextern missing.

### DISCUSSION THREAD

**Discussion by user20112** [emgucv/emgucv#413] [General]
im unsure if im missing a step here but ive referenced emgu.cv.platform.ios and emgu.cv.platform.netstandard and any emgu call is erroring out with system.dllnotfoundexception cvextern.

**Comment by emgucv** [comment]
Please see detail instructions here for the 4.5.1 release:
https://www.emgu.com/wiki/index.php/Download_And_Installation#iOS

Older version such as 4.0 release may have native iOS binary that is no longer compatible with the latest version of XCode.

You also need to use the commercial release, the opensource version do not comes with iOS native binary. You should call `CvInvokeIOS.Init()` function before making any other calls to Emgu CV.

**Reply by user20112** [reply]
Alrighty ill try updating to a newer version ( i need to regain access to the download link which requires going through my it department)

**Reply by user20112** [reply]
Hey ive found the issue it was a mix of old Emgu version having a hard time loading cvextern and me forgetting to preserve a function and it getting linked out in release. Thank you for your time and i hope your day goes well!

**Reply by emgucv** [reply]
That's great. Good to know you have sorted it out.
